packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -174,7 +187,7 @@ function initDataviz(channel) {
       const size = sizeScale(info.neighbors.length);
       const { fill, border } = getNodeColor(node.id, model);
       const fontSize = model.style.fontSize / globalScale;
-      let textColor = d3.rgb(fill);
+      let textColor = fill.toString();
```

Comment:
do I understand correctly that colors are now d3 objects, and only at the final rendering stage they are being converted to strings? I like this

Reply:
Yes - d3.rgb() seems to strip away any opacity for some reason, but returning d3 objects in getNodeColor() and then calling .toString() on them seems to preserve the alpha channel.